### PR TITLE
KeyError seems to be the error we'd like to trap here

### LINKED ISSUE #11: Examples not using default reconciliation service
Hi there, thanks for this project!  Might it be useful for the README to have at least one example using a reconciliation service other than the default one?

Also, I think exposing the [data extension](https://reconciliation-api.github.io/specs/latest/#data-extension-service)  service through this API might also be useful.

### PR DESCRIPTION
**Pull request by gitonthescene**
closes #11

See issue.

### CODE REVIEW

File: reconciler/webutils.py [hunk 1]
```diff
@@ -121,7 +121,7 @@ def parse_raw_results(input_keys, response):
                 current_df.drop(["features"], axis=1, inplace=True)
                 current_df["type_id"] = [item[0]["id"] for item in current_df["type"]]
                 current_df["type"] = [item[0]["name"] for item in current_df["type"]]
-            except IndexError:
+            except KeyError:
```
Comment (jvfe) [inline reconciler/webutils.py]: As discussed on the issue, this would be better:

```suggestion
            except (IndexError, KeyError):
```

**Review by jvfe — CHANGES_REQUESTED** [review]
See the comment I left above.

Other than that, if you want to add an example to the README of another reconciliation endpoint, as described in your issue, it would be very welcome. But don't worry if you don't want to.

File: reconciler/webutils.py (hunk 1 above)
Comment (gitonthescene) [reply reconciler/webutils.py]: Thanks.  I've made this change.  There seems to be a white space change that sneaked in there.  If you'd like I can try to remove it, but it's in the doc.

**Comment by gitonthescene** [comment]
I've made a change to the README to list out the options available.  That should satisfy my purposes.

FWIW, I had another thought.  You can make the tests a little more self-contained if you ran your own service.  It's pretty easy to set one up with [csv-reconcile](https://github.com/gitonthescene/csv-reconcile).  (Full disclosure:  It's my project.  ;-))  You could make a [pytest fixture](https://docs.pytest.org/en/latest/how-to/fixtures.html) to set one up and tear one down per test for isolation.  With some creativity you could make a different service per test.

The tests which connect to the Wikidata reconciliation service can be slow if that server is loaded.  Just an idea ...

Thanks again.

**Review by jvfe — APPROVED** [review]
LGTM Thank you!

**Comment by jvfe** [comment]
> I've made a change to the README to list out the options available. That should satisfy my purposes.
> 
> FWIW, I had another thought. You can make the tests a little more self-contained if you ran your own service. It's pretty easy to set one up with [csv-reconcile](https://github.com/gitonthescene/csv-reconcile). (Full disclosure: It's my project. ;-)) You could make a [pytest fixture](https://docs.pytest.org/en/latest/how-to/fixtures.html) to set one up and tear one down per test for isolation. With some creativity you could make a different service per test.
> 
> The tests which connect to the Wikidata reconciliation service can be slow if that server is loaded. Just an idea ...
> 
> Thanks again.

Oh, that's actually a great idea, would make the test suite much more self-contained and easier to run. I recently found out about csv-reconcile, great project, congrats! I'll keep that idea for the future, thanks.

**Pull request merged**